Modules/Editor/FixturePropertyEditor/ViewModels/FunctionTypeViewModel.cs

```diff
@@ -34,14 +34,167 @@ public FunctionTypeViewModel(Tuple<List<FixtureFunction>, string, Action> functi
+			_timelineColors.Add(Color.LightPink);
+			_timelineColors.Add(Color.LightSalmon);
+			_timelineColors.Add(Color.LightSeaGreen);
+			_timelineColors.Add(Color.LightSkyBlue);
+			_timelineColors.Add(Color.LightSlateGray);
+			_timelineColors.Add(Color.LightSteelBlue);
+			_timelineColors.Add(Color.LightYellow);
+			_timelineColors.Add(Color.Lime);
+			_timelineColors.Add(Color.LimeGreen);
+			_timelineColors.Add(Color.Linen);
+			_timelineColors.Add(Color.Magenta);
+			_timelineColors.Add(Color.Maroon);
+			_timelineColors.Add(Color.MediumAquamarine);
+			_timelineColors.Add(Color.MediumBlue);
+			_timelineColors.Add(Color.MediumOrchid);
+			_timelineColors.Add(Color.MediumPurple);
+			_timelineColors.Add(Color.MediumSeaGreen);
+			_timelineColors.Add(Color.MediumSlateBlue);
+			_timelineColors.Add(Color.MediumSpringGreen);
+			_timelineColors.Add(Color.MediumTurquoise);
+			_timelineColors.Add(Color.MediumVioletRed);
+			_timelineColors.Add(Color.MidnightBlue);
+			_timelineColors.Add(Color.MintCream);
+			_timelineColors.Add(Color.MistyRose);
+			_timelineColors.Add(Color.Moccasin);
+			_timelineColors.Add(Color.NavajoWhite);
+			_timelineColors.Add(Color.Navy);
+			_timelineColors.Add(Color.OldLace);
+			_timelineColors.Add(Color.Olive);
+			_timelineColors.Add(Color.OliveDrab);
+			_timelineColors.Add(Color.Orange);
+			_timelineColors.Add(Color.OrangeRed);
+			_timelineColors.Add(Color.Orchid);
+			_timelineColors.Add(Color.PaleGoldenrod);
+			_timelineColors.Add(Color.PaleGreen);
+			_timelineColors.Add(Color.PaleTurquoise);
+			_timelineColors.Add(Color.PaleVioletRed);
+			_timelineColors.Add(Color.PapayaWhip);
+			_timelineColors.Add(Color.PeachPuff);
+			_timelineColors.Add(Color.Peru);
+			_timelineColors.Add(Color.Pink);
+			_timelineColors.Add(Color.Plum);
+			_timelineColors.Add(Color.PowderBlue);
+			_timelineColors.Add(Color.Purple);
+			_timelineColors.Add(Color.Red);
+			_timelineColors.Add(Color.RosyBrown);
+			_timelineColors.Add(Color.RoyalBlue);
+			_timelineColors.Add(Color.SaddleBrown);
+			_timelineColors.Add(Color.Salmon);
+			_timelineColors.Add(Color.SandyBrown);
+			_timelineColors.Add(Color.SeaGreen);
+			_timelineColors.Add(Color.SeaShell);
+			_timelineColors.Add(Color.Sienna);
+			_timelineColors.Add(Color.Silver);
+			_timelineColors.Add(Color.SkyBlue);
+			_timelineColors.Add(Color.SlateBlue);
+			_timelineColors.Add(Color.SlateGray);
+			_timelineColors.Add(Color.Snow);
+			_timelineColors.Add(Color.SpringGreen);
+			_timelineColors.Add(Color.SteelBlue);
+			_timelineColors.Add(Color.Tan);
+			_timelineColors.Add(Color.Teal);
+			_timelineColors.Add(Color.Thistle);
+			_timelineColors.Add(Color.Tomato);
+			_timelineColors.Add(Color.Turquoise);
+			_timelineColors.Add(Color.Violet);
+			_timelineColors.Add(Color.Wheat);
+			_timelineColors.Add(Color.White);
+			_timelineColors.Add(Color.WhiteSmoke);
+			_timelineColors.Add(Color.Yellow);
+			_timelineColors.Add(Color.YellowGreen);
```

Comment:
Instead of hardcoding all the colors except transparent you should be able to use something like this:

_timelineColors = ((KnownColor[])Enum.GetValues(typeof(KnownColor)))
				.Select(Color.FromKnownColor)
				.Where(color => color != Color.Transparent)
				.ToList();